libpod/reset.go

```diff
@@ -77,18 +77,35 @@ func (r *Runtime) Reset(ctx context.Context) error {
 		}
 	}
 
+	xdgRuntimeDir := os.Getenv("XDG_RUNTIME_DIR")
 	_, prevError := r.store.Shutdown(true)
-	if err := os.RemoveAll(r.store.GraphRoot()); err != nil {
+	graphRoot := r.store.GraphRoot()
+	if graphRoot == xdgRuntimeDir {
```

Comment:
I'm not very familiar with Go, so is that a string equality, or does it handle minor differences between paths as they might have been defined, eg. with or without a trailing `/`.

Reply:
Just string equality.

Reply:
The filepath.Clean() will cleanup extra "/" and other cruft in the paths.